### ISSUE TITLE
API documentation incorrectly specifies base64-encoding

### ISSUE THREAD

**Issue by brianmartin** [kubeflow/pipelines#2412]
**What happened:**

API documentation is incorrect. It specifies that filter.proto json should be base64 encoded. In actuality, it is just required to be URL-encoded (see: https://github.com/kubeflow/pipelines/blob/9d69e8a113cfe8066d85a01101e0b62386ca4192/backend/src/apiserver/server/list_request_util.go#L164-L176).

The wording on the documentation site is:

```
filter | A base-64 encoded, JSON-serialized Filter protocol buffer (see filter.proto).
```
([src](https://www.kubeflow.org/docs/pipelines/reference/api/kubeflow-pipeline-api-spec/#operation--apis-v1beta1-runs-get))

**What did you expect to happen:**

Documentation should not specify base64 encoding.

**Comment by rmgogogo** [comment]
Hi Jing, it seems a quick fix.

**Comment by neuromage** [comment]
Fixed by #2447
